remote_files/docker-compose.yml

```diff
@@ -7,7 +7,12 @@ services:
 
   minitwitimage:
     image: aguldborg/minitwit
```

Comment:
Added as build argument similar to how we set rack and app env

Reply:
Disregard my above comment. We set rack and app env for the docker build but we need to set the version when we pull it instead. So I passed it as a variable to the `./deploy.sh` script.

NB: Lots of the failed attempts happened since I did not synchronize the newest version of our deploy script through vagrant but now it works, see below taking by running `docker ps` on the host machine:
![image](https://github.com/git-gurus-itu-devops/itu-minitwit/assets/95026056/162cab92-0789-4fb1-a917-3a9a7a2afe06)

I also learned that the digitalocean plugin for vagrant comes with the command `vagrant rebuild <name>` which is equivalent to `destroy` and `up` (including synchronizing files) but without changing IP and other configurations made on their online platform.